.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -0,0 +1,63 @@
+name: Deploy standalone plugins to WordPress.org
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'release/**'
+    paths:
+      - '.github/workflows/deploy-standalone-plugins.yml'
+  pull_request:
+    branches:
+      - trunk
+      - 'release/**'
+      - 'feature/**'
+    paths:
+      - '.github/workflows/deploy-standalone-plugins.yml'
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  release:
+    name: New Release
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Set matrix
+        id: set-matrix
+        # Load the JSON file and parse from "{name: {slug, version}, ...}" to "include: [{ name, slug, version }, ...]"
+        # for use in the matrix.
+        run: echo "matrix="$(jq -c '{include:[keys[] as $k | {name:$k,slug:.[$k].slug,version:.[$k].version }]}' plugins.json) >> $GITHUB_OUTPUT
+  deploy:
+    needs: release
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{ fromJSON(needs.release.outputs.matrix) }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Node.js (.nvmrc)
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+      - name: Install npm dependencies
+        run: npm ci
+      - name: Building standalone plugins
+        run: npm run build-plugins
+      - name: Deploy Standalone Plugin - ${{ matrix.slug }}
+        #uses: 10up/action-wordpress-plugin-deploy@stable
+        env:
+          #SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          #SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          SVN_PASSWORD: SVN_PASSWORD
+          SVN_USERNAME: SVN_USERNAME
```

Comment:
Why are these two lines commented out?

Reply:
Because if somehow the workflow starts working, then it will not commit the code in SVN because of wrong access. I'll update it once everything is finalised. Thanks.